config/README.md

```diff
@@ -168,6 +168,8 @@ To configure the default behaviour of individual components (e.g. plugins, ui-wi
  - Port the server is hosted from.
 - `config.server.handle = null`
  - Optional handle object passed to [server.listen](https://nodejs.org/api/http.html#http_server_listen_handle_callback) (aligning port must still be given).
+- `config.server.timeout = 0`
```

Comment:
It's not being disable since it's not > 0. However we should allow disabling too.